src/rules/function-space-after/index.js

```diff
@@ -0,0 +1,45 @@
+import {
+  ruleMessages,
+  isWhitespace
+} from "../../utils"
+
+export const ruleName = "function-space-after"
+
+export const messages = ruleMessages(ruleName, {
+  expected: "Expected single space after function's \")\"",
+  rejected: "Unexpected space after function's \")\"",
+})
+
+const acceptableNextChars = [ " ", ")", undefined ]
+
+/**
+ * @param {"always"|"never"} expectation
+ */
+export default function (expectation) {
+  return function (css, result) {
+    css.eachDecl(function (decl) {
+      if (decl.prop === "content") { return }
+
+      const value = decl.value
+
+      for (let i = 0, l = value.length; i < l; i++) {
+        if (value[i] === ")") {
```

Comment:
No, did not think about looking for something like that (thanks for pointing it out)... At the same time, I'm not sure if this current approach would have bugs that using something like balanced-match would fix. Can you think of any tests that might show that it is necessary, and just looking for closing parens is not enough?